Diary/src/main/java/com/microsoft/device/display/samples/diary/ui/pages/CalendarPage.kt

```diff
@@ -0,0 +1,125 @@
+import java.time.LocalDate
+
+/**
+ *  The CalendarView Composable which also contains the Diary Content for the selected day
+ */
+@Composable
+fun TwoPaneScope.CalendarPage(
+    content: String,
+    selectedDate: LocalDate,
+    updateDate: (LocalDate) -> Unit,
+    updateContent: () -> Unit
+) {
+    val twoPaneScope = this
+    Scaffold(
+        topBar = {
+            TopAppBar(
+                title = {
+                    Text(text = stringResource(id = R.string.app_name))
+                },
+                contentColor = Color.White,
+                backgroundColor = MaterialTheme.colors.primaryVariant,
+                actions = {
+                    if (twoPaneScope.isSinglePane) {
+                        IconButton(
+                            onClick = { twoPaneScope.navigateToPane2() }
+                        ) {
+                            Icon(
+                                imageVector = Icons.Default.Edit,
+                                contentDescription = "Edit diary"
+                            )
+                        }
+                    }
+                }
+            )
+        }
+    ) {
+
+        Column {
+            AndroidView(
+                {
+                    CalendarView(it)
+                        .apply {
+                            layoutParams = ViewGroup.LayoutParams(
+                                ViewGroup.LayoutParams.MATCH_PARENT,
+                                ViewGroup.LayoutParams.WRAP_CONTENT
+                            )
+                        }
+                },
+                modifier = Modifier.wrapContentWidth(),
+                update = {
+                    it.setOnDateChangeListener { _, year, month, dayOfMonth ->
+                        val currentSelectedDate = LocalDate.of(year, month + 1, dayOfMonth)
+                        updateDate(currentSelectedDate)
+                        updateContent()
+                    }
+                }
+
+            )
+
+            Spacer(
+                modifier = Modifier
+                    .background(color = Color.Gray)
+                    .height(1.dp)
+                    .fillMaxWidth()
+            )
+
+            Text(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(all = 10.dp),
+                text = selectedDate.toString(),
+                style = TextStyle(
+                    fontSize = 24.sp,
+                    textAlign = TextAlign.Center,
+                    shadow = Shadow(
+                        color = Color.Gray,
+                        offset = Offset(4.0f, 5.0f),
+                        blurRadius = 2f
+                    )
+                )
+            )
+
+            Text(
+                text = content,
+                Modifier.padding(10.dp),
+                style = TextStyle(
+                    fontSize = 18.sp,
+                    textAlign = TextAlign.Left,
```

Comment:
If you plan on using some of these text styles in multiple places, it's easier to add them to the MaterialTheme (via the Type.kt file) and just call `MaterialTheme.typography.body1` or whichever style you override